include/common.h

```diff
@@ -1249,14 +1252,15 @@ extern void (*D_8010EB98[])();
 extern void (*D_8010EBA0[])();
 extern u8 need_palette_load;
 extern void (*D_8010EBA8[])();
+extern u8 D_801721B8;
 extern s8 D_801721F7;
 extern void (*D_8010FC90[])();
 extern s32 D_8010B1F8[];
 extern s32 D_8010B23C[][4];
-extern s32 D_8013E188[];
-extern s32 D_8013E18C;
-extern s32 D_8013E190;
-extern s32 D_8013E194;
+extern s32 D_8013E188[4];
+// extern s32 D_8013E18C;
+// extern s32 D_8013E190;
+// extern s32 D_8013E194;
```

Comment:
I kept these around as comments so I don't re-add them by accident and instead use the array